mitxgraders/exceptions.py

```diff
@@ -0,0 +1,28 @@
+"""
+exceptions.py
+
+Contains generic grader-related exceptions
+"""
+
+class ConfigError(Exception):
```

Comment:
I think `ConfigError` and `StudentFacingError` should be subclasses of `MITXGraderError`.